joss_paper/paper.md

```diff
@@ -0,0 +1,228 @@
+For statistical validity, the accumulated samples should be collected from equilibrated simulations and *alchemlyb* contains tools for assessing (`alchemlyb.convergence`) and plotting (`alchemlyb.visualisation`) the convergence of the free energy estimate as a function of simulation time [@yang2004free] and means to compute the "fractional equilibration time" [@fan2020aa] to detect potentially non-equilibrated data.
+
+*alchemlyb* offers all these tools as a library for users to customize each stage of the analysis (\autoref{fig:buildingblocks}).
+
+![The building blocks of *alchemlyb*. Raw data from simulation packages are parsed into common data structures depending on the free energy quantities, pre-processed, and processed with a free energy estimator. The resulting free energy differences are analyzed for convergence and plotted for quality assessment.\label{fig:buildingblocks}](Fig1.pdf)
+
+
+## Workflows
+
+The building blocks are sufficient to compute free energies from alchemical free energy simulations and assess their reliability.
+This functionality is used, for example, by the Streamlined Alchemical Free Energy Perturbation (SAFEP) analysis scripts [@Salari2018; @santiagomcrae2023].
+
+*alchemlyb* also provides a structure to combine the building blocks into full end-to-end workflows (module `alchemlyb.workflows`).
+As an example, the `ABFE` workflow for absolute binding free energy estimation reads in the raw input data and performs decorrelation, estimation, and quality plotting of the estimate.
+It can directly estimate quantities such as solvation free energies and makes it easy to calculate more complex quantities such as absolute binding free energies (as the difference between the solvation free energy of the ligand in water and the solvation free energy of the ligand in the protein's binding pocket).
+
+
+# Acknowledgements
+
+Some work on *alchemlyb* was supported by grants from the  National
+Institutes of Health (Award No R01GM118772 to O.B., R35GM148236 to
+D.M., K08GM139031 to T.T.J.) and the National Science Foundation (award ACI-1443054 to O.B.). A.S. acknowledges funding by Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy - EXC 2075 – 390740016 and support by the Stuttgart Center for Simulation Science (SimTech).
+The sponsors were not involved in any aspects of the research or the writing of the manuscript.
+
+We thank Dominik Wille and Travis Jensen for substantial code contributions, Helmut Carter and Wei-Tse Hsu for small fixes, Shujie Fan for initial code for fractional equilibration time calculation, and Jan Janssen for creating the initial conda-forge package.
```

Comment:
@jaclark5 may we list you in the Acknowledgements of the paper?
```suggestion
We thank Dominik Wille, Travis Jensen, and Jennifer A. Clark for substantial code contributions, Helmut Carter and Wei-Tse Hsu for small fixes, Shujie Fan for initial code for fractional equilibration time calculation, and Jan Janssen for creating the initial conda-forge package.
```

Reply:
Yes please! Thank you.